.travis.yml

```diff
@@ -11,20 +11,14 @@ sudo: false # if false, use TravisCI's container based build
 # someday add os: osx to specify osx, but python not yet supported on Travis
 matrix:
     include:
-    - python: 2.7
-      env: CONDA_ENV=py27-min
-    - python: 2.7
-      env: CONDA_ENV=py27
-    - python: 3.4
-      env: CONDA_ENV=py34
     - python: 3.5
       env: CONDA_ENV=py35
     - python: 3.6
       env:
         - CONDA_ENV=py36
         - TASK="coverage"
         - DEPLOY_ENV="true"
-    - python: 3.6
+    - python: 3.7
```

Comment:
@wholmgren Do you know if this is a significant problem in the current builds?

Reply:
It is not a problem -- this only specifies the base travis python version. The conda requirements file specifies the python version that's actually used in the test environment. I figured it was better to use the same version for consistency. But Travis did not yet have a base 3.7 available when it came time to add it to build matrix, so I used 3.6.

Reply:
Can you check my update in 6207754?

It's based on https://github.com/travis-ci/travis-ci/issues/9815.

Reply:
Also, I missed the memo on this: Is the plan to retire Travis once the Azure pipelines are deemed stable?

Reply:
> Can you check my update in 6207754?

I don't know what if any nuance there might be on this, but if it works and it's not noticeably slower then it's ok with me. Is there any benefit to making this change?

> Is the plan to retire Travis once the Azure pipelines are deemed stable?

Main purpose of adding azure was that it let us remove appveyor. I want to see the azure pipelines documentation improve before making a plan to retire Travis.

Reply:
I just set up CI using Azure pipelines for Ubuntu 16.04, Windows, and macOS [here](https://github.com/markcampanelli/pvfit-m/blob/master/azure-pipelines.yml). It might be nice to maintain only a single CI in pvlib.

Reply:
Thanks for the example! #719 tracks this - well, at least the macOS part. Additional work would be required to transition the deploy section of the travis config to azure. #727 also relevant.